src/components/feedback/SpinnerOverlay.js

```diff
@@ -0,0 +1,25 @@
+import Overlay from '../layout/Overlay';
+import Spinner from './Spinner';
+
+/**
+ * @typedef {Omit<import('preact').JSX.HTMLAttributes<HTMLElement>, 'className'>} HTMLAttributes
+ */
+
+/**
+ * Render a full-screen spinner atop a light-colored overlay
+ *
+ * @param {HTMLAttributes} props
+ */
+const SpinnerOverlayNext = function SpinnerOverlay({ ...HTMLAttributes }) {
```

Comment:
```suggestion
const SpinnerOverlayNext = function SpinnerOverlay({ ...htmlAttributes }) {
```

Per our naming conventions, acronyms are lower cased if the first word of an identifier.

Reply:
Good catch, thanks!